activity/history/v0.1.0/schema.json

```diff
@@ -0,0 +1,74 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://common.schemas.verida.io/activity/history/v0.1.0/schema.json",
+    "title": "History",
+    "titlePlural": "Histories",
+    "description": "General history of activities such as listening, watching, browsing, etc.",
+    "type": "object",
+    "appearance": {
+        "style": {
+            "color": "#4A90E2",
+            "icon": "./icon.svg"
+        }
+    },
+    "database": {
+        "name": "activity_history",
+        "indexes": {
+            "name": ["name", "activityType", "timestamp"],
+            "insertedAt": ["insertedAt", "timestamp"],
```

Comment:
Sorry, these should be two separate indexes:

"insertedAt": ["insertedAt"]
"timestamp": ["insertedAt"]

Reply:
> Sorry, these should be two separate indexes:
> 
> "insertedAt": ["insertedAt"] "timestamp": ["insertedAt"]

```
"insertedAt": ["insertedAt"]
"timestamp": ["timestamp"]
```